datacats/cli/purge.py

```diff
@@ -45,9 +45,8 @@ def purge(opts):
     environment.purge_data(sites)
 
     if opts['--delete-environment']:
-        if not environment.target:
+        if environment.target:
+            rmtree(environment.target)
+        else:
             print 'Failed to load environment.',
```

Comment:
I beilieve that is the only one left. All the other cases (see [#153](https://github.com/datacats/datacats/issues/153)) has been fixed since then